aws/tfvars/filecoin-glif-mainnet-apn1.tfvars

```diff
@@ -21,6 +21,7 @@ git_configuration = [
       project_name      = "cid-checker",
       name              = "cid-checker"
       organization_name = "glifio",
+      environment       = "production",
```

Comment:
we don't have a production environment.
dev and mainnet only.

Reply:
fixed